assets/semi-structured/clickhouse/clear-cache-script.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+clickhouse-client --query 'SYSTEM DROP UNCOMPRESSED CACHE'
+clickhouse-client --query 'SYSTEM DROP MARK CACHE'
```

Comment:
We generally prefer to use double-quotes for strings unless there's a reason not to (e.g., to avoid shell interpretation of $).